src/platform/intel/cavs/include/cavs/lib/pm_memory.h

```diff
@@ -25,7 +25,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#define MEMORY_POWER_DOWN_DELAY 256
```

Comment:
beyond clarifying why this needs to be a power of 2, adding a comment on the unit would help